components/list/list-item-drag-handle.js

```diff
@@ -181,25 +182,6 @@ class ListItemDragHandle extends LocalizeCoreElement(FocusMixin(RtlMixin(LitElem
 		this.activateKeyboardMode();
 	}
 
-	_onKeyboardButtonFocusIn() {
```

Comment:
I just renamed these handlers for consistency. I vaguely recall not doing it previously in effort to keep a previous PR smaller.